ci-operator/step-registry/openshift/manifests/runc/openshift-manifests-runc-ref.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "openshift/manifests/runc/openshift-manifests-runc-ref.yaml",
+	"owners": {
+		"approvers": [
+			"harche",
+			"mrunalp",
+			"giuseppe"
```

Comment:
I'm guessing @giuseppe was here because these are being copied from crun jobs. He's welcome to stay but I'm guessing he isn't as concerned about these. can you add @saschagrunert and me as well here?